perllib/FixMyStreet/Cobrand/UK.pm

```diff
@@ -458,6 +458,28 @@ sub updates_disallowed {
     return '';
 }
 
+# Allow cobrands to prevent reports from being reopened
+sub reopening_disallowed {
+    my ($self, $problem) = @_;
+    my $c = $self->{c};
+
+    # Check if reopening is disallowed by the problem's category
+    return 1 if $self->next::method($problem);
+
+    my $cfg = $self->feature('reopening_allowed') || '';
+    if ($cfg eq 'none') {
+        return 1;
+    } elsif ($cfg eq 'staff') {
+        # Only staff and superusers can reopen
+        my $staff = $c->user_exists && $c->user->from_body && $c->user->from_body->name eq $self->council_name;
+        my $superuser = $c->user_exists && $c->user->is_superuser;
+        return 1 unless $staff || $superuser;
+    }
```

Comment:
Would be good to factor the shared bits of this method out to avoid duplication. Currently this version doesn't handle the "reporter" config at all.